packages/@webex/test-helper-mock-webex/src/index.js

```diff
@@ -25,6 +25,7 @@ const nonInternalPlugins = [
   'teams',
   'teamMemberships',
   'webhooks',
+  'presence',
```

Comment:
I dont think we need this in beta. @mkesavan13 can you please confirm this.

Reply:
Yes. This is also the presence plugin change. So, we can revert this too.

Reply:
Reverted